### PR TITLE
Further updates to TLS Bootstrapping

### PR DESCRIPTION
**Pull request by deitch**
More updates per @liggitt comments on https://github.com/kubernetes/website/pull/11181

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md [hunk 1]
```diff
@@ -123,12 +123,12 @@ of provisioning.
...
 
-Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user in the `system:bootstrappers` group. This group
-serves two purposes:
+Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user with the rights to:
 
-1. Will be granted rights to create and retrieve CSRs.
-2. Used in the controller-manager configuration to scope
-approvals in the default approval controller, if automatic approval is enabled. 
+1. create and retrieve CSRs
+2. Be recognized by the controller-manager configuration to scope approvals in the default approval controller, if automatic approval is enabled.
```
Comment (liggitt) [inline content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md]: suggest: "be automatically approved to request node client certificates"

File: content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md [hunk 2]
```diff
@@ -123,12 +123,12 @@ of provisioning.
...
-serves two purposes:
+Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user with the rights to:
 
-1. Will be granted rights to create and retrieve CSRs.
-2. Used in the controller-manager configuration to scope
-approvals in the default approval controller, if automatic approval is enabled. 
+1. create and retrieve CSRs
+2. Be recognized by the controller-manager configuration to scope approvals in the default approval controller, if automatic approval is enabled. 
+
+A kubelet authenticating usinng bootstrap tokens is recognized as a group `system:bootstrappers`, which is the standard method to use.
```
Comment (liggitt) [inline content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md]: s/usinng/using/

"is authenticated as a user in the `system:bootstrappers` group"

**Comment by liggitt** [comment]
Re: https://github.com/kubernetes/website/pull/11181#discussion_r236332597, I would still prefer to trim down the kube-proxy section, or make it more generically discuss running other components on the node, and give kube-proxy as an example. I also don't think we should mention using the kubelet's credentials for other components

File: content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md (hunk 1 above)
Comment (deitch) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md]: Sure.

File: content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md (hunk 2 above)
Comment (deitch) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md]: Want to guess which letter is misbehaving in my MacBookPro keyboard this week?

Sure.

**Comment by deitch** [comment]
Updated per comments.

**Comment by Bradamant3** [comment]
/approve

but see also #11381 -- we need to do still more work on certs generally

**Pull request merged**